Monika After Story/game/script-topics.rpy

```diff
@@ -3587,6 +3587,35 @@ label monika_chloroform:
     m 1tsb "You're so cute when you're nervous."
     return
 
+init 5 python:
+    if os.path.isfile(os.path.expandvars("%APPDATA%") + '\RenPy\JustYuri\persistent'):
+      addEvent(Event(persistent.event_database,eventlabel="just_yuri",category=['club members'],prompt="Just Yuri mod",random=True))
+        
+label just_yuri:
+       m 1esc "[player]..."
+       m 4esc "A thing drew to my attention."
+       m 4esc "Did you..."
+       m 4eka "Did you installed ''Just Yuri'' mod?"
```

Comment:
Replace "installed" with "install".

Remove one set of ' from the two sets surrounding "Just Yuri".